services/webhook/tests/test_review_latency.py

```
@@ -0,0 +1,64 @@
+        TrialResult(1, "small", "cave", 0.2, 3.0, True, False, 100, 50),
+        TrialResult(1, "small", "cave", None, 2.0, True, False, 100, 50),
+        TrialResult(2, "medium", "cave", 0.5, 4.0, False, False, 200, 80),
+        TrialResult(2, "medium", "cave", None, 9.0, False, True, 200, 0),
+    ]
+    report = summarize_trials(trials)
+    assert len(report.slices) == 2
+    c1 = next(s for s in report.slices if s.concurrency == 1)
+    assert c1.n == 3
+    assert c1.errors == 0
+    assert c1.p50_complete_s == 2.0
+    assert c1.p50_ttft_s == 0.1 or c1.p50_ttft_s == 0.2
```

Ruff (0.15.21)

[warning] Consider merging multiple comparisons: `c1.p50_ttft_s in {0.1, 0.2}`.

Merge multiple comparisons

(PLR1714)